### DISCUSSION TITLE
[blocks]: multistep form

### DISCUSSION THREAD

**Discussion by joshwisehub** [shadcn-ui/ui#3163] [Blocks Request]
### Description

A stepper block with either the same components in each step or with forms in each step.

### Example

https://vuetifyjs.com/en/components/steppers/#non-linear

**Comment by premiare** [comment]
Is this what you're after? https://github.com/shadcn-ui/ui/pull/318

**Comment by damianricobelli** [comment]
As @premiare comments, this PR https://github.com/shadcn-ui/ui/pull/318 resolves your request.
